test/e2e/app-dir/mdx/mdx.test.ts

```diff
@@ -59,6 +59,11 @@ for (const type of [
           const $ = await next.render$('/')
           expect($('h2').text()).toBe('This is a client component')
         })
+
+        it('should work with next/image', async () => {
+          const $ = await next.render$('/image')
+          expect($('img').attr('src')).toBe('/next.svg')
```

Comment:
We should change this test from svg to jpg so we can see that it really is optimizing (since svg is vector so it wont optimize and looks identical to native `<img>`)